_includes/selector.html

```diff
@@ -424,6 +434,25 @@
                 }
                 return pkg_components[0] + channel_pkg_separator + this.highlightPkgOrImg(pkg_components[1]);
             },
+            getCondaVersionSupport(version) {
+                var cuda_version_info = {
+                    "Stable": {
+                        "11": ["11.4", "11.8"],
+                        "12": ["12.0", "12.2"]
+                    },
+                    "Nightly": {
+                        "11": ["11.4", "11.8"],
+                        "12": ["12.0", "12.5"]
+                    }
+                };
```

Comment:
Wonder if we can collect this info next to the "all possible values" section. That way there would be one place to change all of the versions

Reply:
I struggled with this. I don't know why, but I couldn't get the JavaScript to work happily when I did that. See my attempt here (I force-pushed it out of the history for this PR since it wasn't going well): https://github.com/rapidsai/docs/commit/3a6e9a595313af4c1c9b5eb6f816f9c25cc222a0